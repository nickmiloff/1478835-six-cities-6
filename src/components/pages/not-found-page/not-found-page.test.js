import React from 'react';
import {render} from '@testing-library/react';
import Test from '../../test/test';

import NotFoundPage from './not-found-page';

describe(`NotFoundPage component test`, () => {
  it(`'NotFoundPage component' should render correctly`, () => {
    const {container} = render(
        <Test>
          <NotFoundPage />
        </Test>
    );

    expect(container).toMatchSnapshot();
  });
});
