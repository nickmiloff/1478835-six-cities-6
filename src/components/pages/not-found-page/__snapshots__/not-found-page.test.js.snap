// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component test 'NotFoundPage component' should render correctly 1`] = `
<div>
  <div
    class="page page--gray page--favorites-empty"
  >
    <header
      class="header"
    >
      <div
        class="container"
      >
        <div
          class="header__wrapper"
        >
          <div
            class="header__left"
          >
            <a
              class="header__logo-link header__logo-link--active"
              href="/"
            >
              <img
                alt="6 cities logo"
                class="header__logo"
                height="41"
                src="img/logo.svg"
                width="81"
              />
            </a>
          </div>
          <nav
            class="header__nav"
          >
            <ul
              class="header__nav-list"
            >
              <li
                class="header__nav-item user"
              >
                <a
                  class="header__nav-link header__nav-link--profile"
                  href="/login"
                >
                  <div
                    class="header__avatar-wrapper user__avatar-wrapper"
                  />
                  <span
                    class="header__login"
                  >
                    Sign in
                  </span>
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </div>
    </header>
    <main
      style="width: 100%; height: 100%; display: flex; flex-direction: column; justify-content: center; align-items: center;"
    >
      <h1>
        404 Page not found
      </h1>
      <a
        href="/"
      >
        Home page
      </a>
    </main>
  </div>
</div>
`;
